### PR TITLE
Added test for the new options implemented in #9

### PR DESCRIPTION
**Pull request by n1xx1**

### CODE REVIEW

File: src/encoder.rs [hunk 1]
```diff
@@ -587,6 +588,13 @@ pub(crate) mod tests {
         };
 
         ctx.configure().unwrap();
+
+        ctx.set_option("cpu-used", 0u64).unwrap();
```
Comment (lu-zero) [inline src/encoder.rs]: maybe try setting non-zero values

File: src/encoder.rs [hunk 2]
```diff
@@ -381,31 +381,31 @@ mod encoder_trait {
                 ("cpu-used", Value::U64(v)) => {
                     self.enc.as_mut().ok_or(Error::InvalidData).and_then(|x| {
                         x.control(vp8e_enc_control_id::VP8E_SET_CPUUSED, v as i32)
-                            .map_err(|_err| Error::InvalidData)
+                            .map_err(|_err| Error::ConfigurationInvalid)
```
Comment (lu-zero) [inline src/encoder.rs]: Sounds a good idea indeed :)

File: src/encoder.rs (hunk 1 above)
Comment (n1xx1) [reply src/encoder.rs]: done :)

**Comment by lu-zero** [comment]
Thank you!

**Pull request merged**